src/main/java/org/stellar/sdk/responses/AssetResponse.java

```diff
@@ -24,15 +24,31 @@ public class AssetResponse extends Response implements Pageable {
   @SerializedName("contract_id")
   String contractID;
 
-  @SerializedName("accounts")
-  AssetResponse.Accounts accounts;
+  // TODO: The following fields will be removed in Horizon 3.0,
+  //  and we should also remove them at that time.
+  @SerializedName("num_accounts")
+  int numAccounts;
 
-  @SerializedName("balances")
-  AssetResponse.Balances balances;
+  @SerializedName("num_claimable_balances")
+  int numClaimableBalances;
+
+  @SerializedName("num_liquidity_pools")
+  int numLiquidityPools;
+
+  @SerializedName("num_contracts")
+  int numContracts;
+
+  @SerializedName("num_archived_contracts")
+  int numArchivedContracts;
+
+  // TODO: remove the above fields when Horizon 3.0 is released.
```

Comment:
not sure what 3.0 will look like, maybe can remove this, tackle it later when it's known?

Reply:
I saw that [relevant comment](https://github.com/stellar/go/blob/master/protocols/horizon/main.go#L171-L177) has been added in stellar/go. But we can remove this TODO first, and then remove it here after horizon confirms its removal.